src/OOVPADatabase/D3D8LTCG/3911.inl

```diff
@@ -695,6 +695,24 @@ OOVPA_NO_XREF(D3DDevice_DrawIndexedVerticesUP, 1024, 10)
         { 0x36, 0x57 },
 OOVPA_END;
 
+// ******************************************************************
+// * D3DDevice_DrawIndexedVertices
+// ******************************************************************
+OOVPA_NO_XREF(D3DDevice_DrawIndexedVertices, 1024, 15)
+
+        // push ebp
+        OV_MATCH(0x00, 0x55),
+
+        // mov eax, [esi+0x0478]
+        OV_MATCH(0x0D, 0x8B, 0x86, 0x78, 0x04, 0x00, 0x00),
+
+        // mov dword ptr [eax], 000417FC
+        OV_MATCH(0x51, 0xC7, 0x00, 0xFC, 0x17, 0x04, 0x00),
```

Comment:
I notice 0x000417FC address is relying on "game" specific detection. We do not use hardcode address in any of our database in order to have better detection across titles.

The reason we do not allow this since xbe's sections can be resize at any time, depending on what APIs are used. Or shift around, such as add/remove sections. No xbes are exactly the same nor sort in same ordering.

Reply:
I based this off the respective signature for a non-LTCG version:
https://github.com/Cxbx-Reloaded/XbSymbolDatabase/blob/master/src/OOVPADatabase/D3D8/3911.inl#L1742

So either this is not an address or the other signature should be corrected too.

Reply:
I check 0x000417FC address is API specific detection. So we do approve this address.

Reply:
My apologies, I should had check from my end first. It is not an actual address reference. 👍